Firestore/Swift/Source/AsyncAwait/Firestore+AsyncAwait.swift

```diff
@@ -53,5 +53,50 @@ import Foundation
         }
       }
     }
+
+    /// Executes the given updateBlock and then attempts to commit the changes applied within an atomic
+    /// transaction.
+    ///
+    /// The maximum number of writes allowed in a single transaction is 500, but note that each usage of
+    /// `FieldValue.serverTimestamp()`, `FieldValue.arrayUnion()`, `FieldValue.arrayRemove()`, or
+    /// `FieldValue.increment()` inside a transaction counts as an additional write.
+    ///
+    /// In the updateBlock, a set of reads and writes can be performed atomically using the
+    /// `FIRTransaction` object passed to the block. After the updateBlock is run, Firestore will attempt
```

Comment:
Drop the FIR prefix.

Reply:
Done, also did this in `FIRFirestore.h` for consistency.